pagerduty/resource_pagerduty_automation_actions_action.go

```diff
@@ -153,6 +158,11 @@ func buildAutomationActionsActionStruct(d *schema.ResourceData) (*pagerduty.Auto
 		automationActionsAction.ModifyTime = &val
 	}
 
+	if attr, ok := d.GetOk("only_invocable_on_unresolved_incidents"); ok {
```

Comment:
`GetOk` will return `ok` equal to `false` even when the value is explicitly set to `false`. Since the client defines this field as `omitempty` it would never get changed to false.
I'd change this whole `if` block to:
```
attr, _ :=  d.Get("only_invocable_on_unresolved_incidents").(bool)
automationActionsAction.OnlyInvocableOnUnresolvedIncidents = &attr
```